orttraining/orttraining/python/training/_ortmodule_logger.py

```diff
@@ -1,21 +1,30 @@
 import os
 import sys
 from contextlib import contextmanager
+from enum import IntEnum
+
+
+class LogLevel(IntEnum):
+    VERBOSE = 0
+    INFO = 1
+    WARNING = 2
+    ERROR = 3
+    FATAL = 4
 
 
 @contextmanager 
-def suppress_output(suppress_stdout=True,suppress_stderr=True, f=os.devnull):
-    """Supress output from being printed to stdout and stderr."""
+def suppress_output(suppress_stdout=True,suppress_stderr=True, f=os.devnull, log_level=LogLevel.WARNING):
```

Comment:
Isn't  `f=os.devnull` an implementation detail?

Reply:
It is. In the future, we might want to do some logging to some file that we can pass in here. So, I let this be configurable and set the default value as ```os.devnull```.

Reply:
ok, let me change so that we don't expose this for now. If we want, we can make this configurable in the future.